frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx

```diff
@@ -9,12 +10,27 @@ import DetailedCourse from './DetailedCourse';
 const DetailedCourseContainer: React.FC = () => {
   const params = useParams();
   const { data } = useGetCourseInfo(params.courseId);
-  const { mutate } = useApplyCourse();
-  const handleApplyCourse = () => {
+  const { mutate, status } = useApplyCourse();
+  const [targetId, setSpinner] = useState<string | undefined>();
```

Comment:
Clarify useState variables naming please

Reply:
Fixed and renamed